### ISSUE TITLE
Installation using virtualenv

### ISSUE THREAD

**Issue by heretogo** [lightbulb-framework/lightbulb-framework#3]
I typically use Python in a per user virtualenv and the install script does not play nicely with this. Is there any way to get lightbulb to work with this type of configuration?

**Comment by heretogo** [comment]
Installed framework using:
`pip install lightbulb-framework`

Output of lightbulb status:

```
+--------------------------+--------+
| Name                     | Status |
+--------------------------+--------+
| six                      | OK     |
| stevedore                | OK     |
| pbr                      | OK     |
| cmd2                     | FAIL   |
| unicodecsv               | FAIL   |
| yaml                     | FAIL   |
| prettytable              | FAIL   |
| cliff                    | OK     |
| multiprocessing          | OK     |
| FAdo                     | OK     |
| dateutil                 | FAIL   |
| symautomata              | OK     |
| sfalearn                 | OK     |
| MySQLdb                  | FAIL   |
| OpenFST Python Extension | FAIL   |
+--------------------------+--------+

```

Install as user fails
Install as sudo suceeds but the lightbulb executable does not find the modules

**Comment by heretogo** [comment]
```
user@debian:~% which pip
/home/user/bin/pip
```

```
user@debian:~% which python
/home/user/bin/python
```

**Comment by istais** [comment]
I wont use the 'activate' command so that the situation is more clear. I did the following test in a new installation of debian:

```
cd ~
virtualenv env
env/bin/pip install lightbulb-framework
env/bin/lightbulb-status
```

and then I installed all python packages as user (and only did sudo in cases where it is necessary)

My status was

```

+--------------------------+--------+
| Name                     | Status |
+--------------------------+--------+
| six                      | OK     |
| stevedore                | OK     |
| pbr                      | OK     |
| cmd2                     | OK     |
| unicodecsv               | OK     |
| yaml                     | OK     |
| prettytable              | OK     |
| cliff                    | OK     |
| multiprocessing          | OK     |
| FAdo                     | OK     |
| dateutil                 | OK     |
| symautomata              | OK     |
| sfalearn                 | OK     |
| MySQLdb                  | FAIL   |
| OpenFST Python Extension | FAIL   |
+--------------------------+--------+

```

I fixed MySQLdb by doing

```
env/bin/pip install MySQL-python
```

Now, about pywrapfst, the problem is the following:

If you do the following it works:

```
/usr/bin/python
import pywrapfst
```

While if you do the following it does not:

```
env/bin/python
import pywrapfst
```

which means that the packages were not installed for the user. It is possible to solve the problem by reinstalling all the python packages without the --user and without sudo, in the virtualenv that you created. Maybe I will add this as an option. 

However, for the openfst, we must find a way to install the bindings (the .so library) in the env folder. This should normally occur by providing the correct LD_LIBRARY and python runtime path at the configuration. I am still checking this

**Comment by istais** [comment]
I think that it is now fixed in 0.0.13. 

```
pip install virtualenv
virtualenv env
source env/bin/activate
pip install lightbulb-framework
lightbulb status
```

If you use virtualenv in linux, the "sudo" command will be required only for the installation of libmysqlclient-dev package.

**Comment by heretogo** [comment]
In the end I ended up having to do sudo install on everything ``lightbulb status`` and it works. I am now having an issue with Flex.
